proposals/templates/shader-model-template.md

```diff
@@ -0,0 +1,140 @@
+<!-- {% raw %} -->
+
+# Feature name
+
+## Instructions
```

Comment:
I wonder if there's some value in something explaining what type of proposal this template is intended for (so, eg, a proposal for a new language feature is redirected to the appropriate template)?